config/rbac/kustomization.yaml

```diff
@@ -4,6 +4,8 @@ resources:
 #- role_binding.yaml
 - log-collector-privileged_role.yaml
 - log_collector_privileged_binding.yaml
+- log-collector-loki-clusterrole.yaml
+- log-collector-loki-clusterrolebinding.yaml
```

Comment:
IMHO these should be reconciled by one of CLO's controllers since we don't need them installed on a cluster w/o using lokistack.

Reply:
Updated the implementation of the controller to reconcile the two resources when a "LokiStack" log store is in use. There's a caveat to this though:

There can be no owner-references from a namespaced object ("ClusterLogging") to a non-namespaced one. This means that the CLO will only remove the ClusterRole/ClusterRoleBinding when the ClusterLogging still exists but the log-store is either removed or updated to not include a LokiStack log store.

I wonder if a Role/RoleBinding would be more fitting for this authorization use-case, after all the LokiStack is a namespaced object as well, so I think it would make sense that the authorization is namespaced as well. I have not checked yet if the gateway also recognises a RoleBinding, but I will do that soon.